src/components/InstrumentPicker/InstrumentListItem.tsx

```diff
@@ -0,0 +1,71 @@
+import {lighten, rem} from 'polished';
+import * as React from 'react';
+import {InstrumentModel} from '../../models/index';
+import Dir from '../../stores/dir';
+import {asChange} from '../../utils';
+import styled from '../styled';
+import {InstrumentPickerActions} from './index';
+
+// tslint:disable-next-line:no-var-requires
+const {Flex} = require('grid-styled');
+
+interface InstrumentListItemProps
+  extends InstrumentModel,
+    InstrumentPickerActions {}
+
+const StyledInstrumentItem = styled(Flex)`
+  cursor: pointer;
+  :hover {
+    background: ${lighten(0.05, '#3c3c3c')};
+  }
+`;
+
+const InstrumentField: React.SFC<{dir?: Dir; className?: string}> = ({
```

Comment:
Could you, plz, move it to the separate interface.

Reply:
ok, done